raiden/network/transport/matrix/transport.py

```diff
@@ -395,6 +395,7 @@ def on_success(greenlet):
         self._client.sync_thread.link_exception(self.on_error)
         self._client.sync_thread.link_value(on_success)
         self.greenlets = [self._client.sync_thread]
+        self._schedule(30, self._address_mgr.get_status_info())
```

Comment:
You're calling the function instead of passing it.

Reply:
Right... didn't make a difference in this case as it has no args anyway right?

Reply:
No, it does! You're calling the function once here and then passing `None` to `_schedule()`

Reply:
Makes perfect sense. I got confused by the logs, which were than only from the single call during start.